pam/pam.go

```diff
@@ -84,6 +113,7 @@ func (h *pamModule) handleAuthRequest(mode authd.SessionMode, mTx pam.ModuleTran
 	// Attach logger and info handler.
 	// TODO
 
+	parsedArgs := parseArgs(args)
```

Comment:
If you are not using this anywhere else, you should just parse it right with the `newClient` call

Reply:
Not here, but this is something that is going to be used at later point, (see for example #233), so I preferred to lave it as a variable not to have to refactor it again at later point.